pages/home.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ana Gabriele</title>
+    <link rel="stylesheet" href="../styles/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Montserrat:ital,wght@0,400;1,700&display=swap" rel="stylesheet">
+  <link href="//db.onlinewebfonts.com/c/0fddb08144cea533dcd3d39388b7ffbe?family=GrandSlang+Roman" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    
+    <header>
+        <nav class="block">
+            <ul class="home-nav">
+                <li><a href="https://twitter.com/ana23961938">GitHub</a></li>
+                <li><a href="https://www.linkedin.com/in/ana-gabriele-singh-789b75101/">LinkendIn</a></li>    
+                <li><a href="https://twitter.com/ana23961938">Twitter</a></li>    
+            </ul>
+        </nav>
+    </header>
+
+    <main class="parent-container">
+        <aside class="grid-item-sidebar ">
+            <ul class="home-sidebar">
+                <li><a href="about.html">About</a></li>
+                <li><a href="projects.html">Projects</a></li>
+                <li><a href="contact.html">Contact</a></li>
+            </ul>
+        </aside>
+        <article class="grid-item-desc description">
+            <h2>
+                ANA GABRIELE SINGH
+            </h2>
```

Comment:
It looks like this page has an `h2` but no `h1`.  The `h1`, `h2`, `h3`, etc tags should be able to make a table of contents.  (This is important for things like [screen readers](https://en.wikipedia.org/wiki/Screen_reader) which let the blind access the web.)